qiskit/pulse/channels/device_specification.py

```diff
@@ -58,13 +58,19 @@ def create_from(cls, backend):
 
         # frequency information
         qubit_lo_freqs = config.defaults['qubit_freq_est']
```

Comment:
This is only in `config` temporarily. It will be should be obtained in `backend.defaults().qubit/meas_freq_est`. Try using this and then temporarily fall back to the `config` if `defaults` request fails for now. Make sure to leave a detailed note for why this behaviour is as it is.

Reply:
I added the change to #44 as lo related modification.